.github/workflows/on_merge_to_main.yml

```diff
@@ -11,6 +11,9 @@ jobs:
   prerelease:
     name: Prerelease test and build
     runs-on: ubuntu-latest
+    env:
+      TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
```

Comment:
These were added to Github per [the docs](https://turborepo.org/docs/ci/github-actions)